Project.toml

```diff
@@ -43,3 +43,10 @@ StatsBase = "0.32, 0.33"
 TableTraits = "0.4, 1"
 Tables = "1"
 julia = "1"
+
+# Required for Julia version < 1.2. 
+[extras]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+
+[targets]
+test = ["Documenter"]
```

Comment:
We explicitly activate `test/Project.toml` on Julia < 1.2 - is it not sufficient to add Documenter to test/Project.toml?

Reply:
That's strange. I did that based on the failing action for commit `fdedc01` and the [test/Project.toml source code](https://github.com/rikhuijzer/MCMCChains.jl/blob/fdedc019a9a446ee9a00efd40cbda9c42120f353/test/Project.toml).

Reply:
Related: I've changed continue-on-error to
```
continue-on-error: ${{ matrix.version == 'nightly' || matrix.os == 'ubuntu-latest' }}
```
because the CI sometimes fails on a test in rstar.jl even though I didn't touch the source code. I hope that allowing the Ubuntu tests to continue will show what's up with that.

Reply:
The discussion at <https://github.com/JuliaLang/Pkg.jl/issues/2097#issuecomment-714316161> seems related. How about keeping the workaround with `[extras]` for a bit? It's likely that 1.6 is gonna be LTS anyway according to https://discourse.julialang.org/t/julia-v1-6-0-beta1-is-now-available/53065/5.

Reply:
I fixed the test setup, the only issue was that you tried to load Documenter in runtests.jl before test/Project.toml was activated.

Reply:
It is not relevant if 1.6 becomes the new LTS (which will also only happen - if it happens - when 1.7 is released), we should always test the oldest version that is supported. I.e. it can only be changed if we drop support for Julia < 1.2.